src/features/input/InputView.vue

```diff
@@ -169,34 +209,50 @@ export default {
         }
       ).addTo(this.map);
     },
-    selectAOIFileJson() {
-      this.aoiInputJson.click();
-    },
-    onChangeAOIInputJson(event) {
-      //onChangeAOIInputJson() {
-      //const [file] = event.target.files;
-      var file = event.target.files[0];
-      this.aoiFile = file;
-    },
-    selectAOIFileGpkg() {
-      this.aoiInputGpkg.click();
+    selectSamplesFile() {
+      this.$refs.samplesFileInput.click();
     },
-    loadGeoPackage(event) {
-      const [file] = event.target.files;
-      this.aoiFile = file;
-      console.log(this.aoiFile);
+    onChangeSamplesFileInput(event) {
+      const file = event.target.files[0];
+      this.samplesFile = file;
     },
+    /**
+     * Checks the form and send it to the API.
+     */
+    async onSubmitForm(e) {
+      // prevent that the form is send:
+      e.preventDefault();
 
-    warn_to_increase_Cloud_Cover() {
-      if (document.getElementbyID("cloud_cover") != "15%") {
-        alert("test");
+      // TODO: Implement a better form check with yup:
+      if (!this.formData.name || !this.samplesFile || !this.drawnItem) {
+        return;
+      }
+
+      // create job object for the api
+      const job = {
+        name: this.formData.name,
+        use_lookup: false,
+        resolution: Number.parseInt(this.formData.resolution, 10) || 10,
+        cloud_cover: Number.parseInt(this.formData.cloud_cover, 10) || 15,
+        start_timestamp: `${this.formData.start_timestamp}T00:00:00.000Z`,
+        end_timestamp: `${this.formData.end_timestamp}T00:00:00.000Z`,
+        samples_class: this.formData.samples_class,
+        sampling_strategy: "regular",
+        use_pretrained_model: false,
+        area_of_interest: {
+          type: "FeatureCollection",
+          features: [{ ...this.aoiJson }],
```

Comment:
Was bedeutet hier das ... vor dem this.aoiJson?

Reply:
Dies ist ein Object Spread Operator.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax